partials/story-postcard.hbs

```diff
@@ -0,0 +1,24 @@
+{{! prettier-ignore }}
+<article class="story-postcard">
+    <div onclick="location.href='{{url}}'" class="flex flex-col cursor-pointer h-full rounded-xl overflow-hidden md:flex-row md:h-60 lg:flex-col lg:h-full">
+        {{#if feature_image}}
+        <img class="object-cover m-0 w-full h-60 md:w-2/5 md:h-full lg:w-full lg:h-60" src="{{img_url feature_image}}" alt="{{title}}" />
+        {{else}} {{!-- Placeholder --}}
+        <img class="object-cover m-0 hidden md:block md:w-2/5 md:h-full lg:w-full lg:h-60" src="https://via.placeholder.com/100/FFFFFF/FFFFFF" alt="{{title}}" />
+        {{/if}}
+        <div class="flex-grow bg-gray-100 px-3 pt-2 pb-4">
+            <div>
+                <p class="text-sm my-1">
+                    {{#if primary_tag}}
+                    {{primary_tag.name}} - 
```

Comment:
Preferably the tag should be displayed underneath the title in the form of pills, something like https://material-ui.com/components/chips/. But I think we can do this in a separate PR.